test/datewithzone.test.ts

```diff
@@ -37,8 +37,9 @@ describe('rezonedDate', () => {
     const currentLocalDate = DateTime.local(2000, 2, 6, 1, 0, 0)
     setMockDate(currentLocalDate.toJSDate())
 
-    const d = DateTime.fromISO('20101005T110000').toJSDate()
+    const d = DateTime.fromISO('20101005T110000Z').toJSDate()
```

Comment:
The purpose of this test is not to test the ability to use UTC (which already works). It's to test conversion from local time into the expected timezone.